OilSandsConfig_en.ts

```diff
@@ -0,0 +1,608 @@
+const config = {
+    introSlide: {
+        logo: {
+            src: "" // TODO
+        },
+        title: "# NPRI Sector Overview: Oil Sands Extraction",

```

Comment:
I think the heading tags `#` are probably not needed for titles

Reply:
removed them!